frontend/src/concepts/pipelines/content/pipelineSelector/PipelineSelectorTableRow.tsx

```diff
@@ -3,39 +3,66 @@ import { Stack, StackItem, Tooltip } from '@patternfly/react-core';
 import { Td, Tr } from '@patternfly/react-table';
 import { ExperimentKFv2, PipelineKFv2, PipelineVersionKFv2 } from '~/concepts/pipelines/kfTypes';
 import { relativeTime } from '~/utilities/time';
+import { isArgoWorkflow } from '~/concepts/pipelines/content/tables/utils';
 
 type PipelineSelectorTableRowProps = {
   obj: PipelineKFv2 | PipelineVersionKFv2 | ExperimentKFv2;
+  isSchedule?: boolean;
+  isCreatePage?: boolean;
   onClick: () => void;
 };
 
-const PipelineSelectorTableRow: React.FC<PipelineSelectorTableRowProps> = ({ obj, onClick }) => {
+function isPipelineVersionKFv2(
+  obj: PipelineKFv2 | PipelineVersionKFv2 | ExperimentKFv2,
+): obj is PipelineVersionKFv2 {
+  return 'pipeline_spec' in obj;
+}
+
+const PipelineSelectorTableRow: React.FC<PipelineSelectorTableRowProps> = ({
+  obj,
+  isSchedule,
+  isCreatePage,
+  onClick,
+}) => {
   const tooltipRef = React.useRef(null);
 
+  const isDisabled = React.useMemo(
+    () => isPipelineVersionKFv2(obj) && isArgoWorkflow(obj.pipeline_spec),
+    [obj],
+  );
+
   return (
     <>
       <Tooltip
         position="right"
         isContentLeftAligned
         content={
-          <Stack hasGutter>
-            <StackItem>
-              Name:
-              <br />
-              {obj.display_name}
-            </StackItem>
-            {obj.description && (
+          isCreatePage && isDisabled ? (
+            `Cannot create ${isSchedule ? 'schedule' : 'run'} for Kubeflow v1 SDK pipelines`
+          ) : (
+            <Stack hasGutter>
               <StackItem>
-                Description:
+                Name:
                 <br />
-                {obj.description}
+                {obj.display_name}
               </StackItem>
-            )}
-          </Stack>
+              {obj.description && (
+                <StackItem>
+                  Description:
+                  <br />
+                  {obj.description}
+                </StackItem>
+              )}
+            </Stack>
+          )
         }
         triggerRef={tooltipRef}
       />
-      <Tr ref={tooltipRef} onRowClick={onClick} isClickable>
+      <Tr
+        ref={tooltipRef}
+        onRowClick={!(isCreatePage && isDisabled) ? onClick : undefined}
+        isClickable={!(isCreatePage && isDisabled)}
+      >
```

Comment:
this does not have the disabled styling

Reply:
Had a [conversation](https://patternfly.slack.com/archives/C4FM977N0/p1721295556345339) with Patternfly team. Currently, there is no method to apply disabled styling on clickable rows. I'll raise a feature request with Patternfly. Until then, we can use `isStriped` if @yannnz agrees??

Disabled rows with `isStriped`
![Screenshot 2024-07-19 at 1 48 32 PM](https://github.com/user-attachments/assets/b229ec4f-a914-45d4-8b93-6986d8222b48)

Reply:
i dont think we should use isStriped. it doesn't seem like it looks disabled still. maybe its better to just hide them if we cant disable properly

Reply:
@yannnz wdyt?

Reply:
+1 to Gage. I think hiding them is a good approach. We use the same strategy for Archived experiment under the Experiment dropdown list under the create Run page.  @dpanshug